hub/core/chunk_engine/write.py

```diff
@@ -14,11 +15,19 @@
 from hub.util.array import normalize_and_batchify_shape
 
 
+def write_tensor_meta(key: str, storage: StorageProvider, meta: dict):
+    storage[get_meta_key(key)] = pickle.dumps(meta)
+
+
+def write_dataset_meta(storage: StorageProvider, meta: dict):
+    storage[constants.META_FILENAME] = pickle.dumps(meta)
+
+
 def write_array(
     array: np.ndarray,
     key: str,
-    chunk_size: int,
     storage: StorageProvider,
+    chunk_size: int = 16_000_000,
```

Comment:
in another PR we are putting this into constants.py so no need to change it here

Reply:
(setting it default is fine here i was just saying that in pytest-options we added more stuff in constants.py that will make this definition clean)